### ISSUE TITLE
Metadata for maps that are scanned resources in Figgy is not displaying under viewer i in DPUL

### ISSUE THREAD

**Issue by kelea99** [pulibrary/figgy#3978]
DPUL User: Gabriel Swift

As a site builder, I would like scanned resources that are maps to display all the metadata that is in figgy in the fields shown below the viewer in DPUL, versus in the viewer itself. 

@tpendragon Question: is similar to the Finding aids request (even though the metadata is already in figgy)? If so, would you like individual tickets for each field needed to map to DPUL?

**Comment by tpendragon** [comment]
@kelea99 Can we get an example URL of something broken?

**Comment by kelea99** [comment]
@tpendragon sure.  https://dpul.princeton.edu/pudl0017/catalog/n296x0848

But there are more here: https://dpul.princeton.edu/pudl0017/browse/historic-maps-and-atlases

**Comment by tpendragon** [comment]
Yeah, similar issue to what was up with finding aids. JSON-LD isn't rendering for Scanned Maps:

https://figgy.princeton.edu/catalog/89cfc9a4-fa94-4f8b-8cc3-195648bfc6e5.jsonld
